include/SFML/Graphics/RenderTexture.h

```diff
@@ -85,6 +85,16 @@ CSFML_GRAPHICS_API void sfRenderTexture_destroy(sfRenderTexture* renderTexture);
 ////////////////////////////////////////////////////////////
 CSFML_GRAPHICS_API sfVector2u sfRenderTexture_getSize(const sfRenderTexture* renderTexture);
 
+////////////////////////////////////////////////////////////
+/// \brief Tell if the render target will use sRGB encoding when drawing on it
+///
+/// \param renderTexture Render texture object
+///
+/// \return sfTrue if the render target use sRGB encoding, false otherwise
```

Comment:
I see some comments use `\return True if ..., false otherwise` but it is more often `\return sfTrue if ..., sfFalse otherwise`.
```suggestion
/// \return sfTrue if the render target use sRGB encoding, sfFalse otherwise
```